exporters/variables-scss/src/formatters/stylesFormatter.ts

```diff
@@ -1,4 +1,5 @@
-const INDENTATION = '    ';
+export const SCSS_INDENTATION = '    ';
```

Comment:
We are mixing the inner and outer constants. What about using a config object like

```js
const formattingConfig = {
  js: {
    indentations: '  ',
    openingBracket: '{',
    closingBracket: '}',
  },
  'scss': {
    // ...
  }
}
```

and then in the function body only assign js/scss based on the `hasJsOutput`?